CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### Release Notes
 
+### Breaking changes
```

Comment:
I don't think it's a good idea to have this enabled with a limit by default. The fact that Breaking Changes would have to appear in the changelog is a red flag to me. We shouldn't have any breaking changes until 2.0.